src/bug-filing/common/get-issue-details-markdown.ts

```diff
@@ -0,0 +1,44 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+import { EnvironmentInfo } from '../../common/environment-info-provider';
+import { CreateIssueDetailsTextData } from '../../common/types/create-issue-details-text-data';
+import { IssueUrlCreationUtils } from './issue-filing-url-string-utils';
+
+export type IssueDetailsGetter = (
```

Comment:
```suggestion
export type IssueDetailsMarkDownGetter = (
```

Reply:
Exploring 'suggested change' feature, I think we should use it more. :-)

Reply:
I agree. One caveat thought: sometimes a rename suggestion may not rename all usages of the symbol being change, resulting in build/test failures.